Documentation/Reference.md

```diff
@@ -341,3 +358,21 @@ A C language target is build with following flags in release mode:
 ### Depending on Apple Modules
 
 At this time there is no explicit support for depending on Foundation, AppKit, etc, though importing these modules should work if they are present in the proper system location. We will add explicit support for system dependencies in the future. Note that at this time the Package Manager has no support for iOS, watchOS, or tvOS platforms.
+
+## C language targets
+
+The C language module are laid out similar to Swift targets execept that the C langauge libraries should contain a directory named `include` to hold the public headers.  
```

Comment:
s/module/targets/.... I would like to move to using target consistently to mean the grouping object, and module only when it is technically correct (a Swift module, or a Clang module).